test/runtests.jl

```diff
@@ -13,10 +13,9 @@ using Adapt
 using KernelAbstractions
 import ReverseDiff as RD        # used in `pooling.jl`
 
-## TODO Enzyme tests are broken at the moment
-# const Test_Enzyme = VERSION <= v"1.10-" && !Sys.iswindows() &&
-#     # TODO Enzyme is not working properly with AMDGPU yet.
-#     get(ENV, "NNLIB_TEST_AMDGPU", "false") != "true"
+const Test_Enzyme = VERSION <= v"1.10-" && !Sys.iswindows() &&
+    # TODO Enzyme is not working properly with AMDGPU yet.
+    get(ENV, "NNLIB_TEST_AMDGPU", "false") != "true"
 const Test_Enzyme = false
```

Comment:
don't you need to also remove this variable?

Reply:
uops